xbmc/network/upnp/UPnPInternal.cpp

```diff
@@ -38,6 +38,28 @@ using namespace XFILE;
 
 namespace UPNP
 {
+// add some hard coded allowed subtitle just like
+// https://github.com/xbmc/xbmc/blob/bdf0f9b468475a4fc4e603eebef004175182968c/xbmc/network/upnp/UPnPInternal.cpp#L663 did
+// see also line#684
+// the original version of content type here,eg: text/srt,which was defined 10 years ago (year 2013,
+// https://github.com/xbmc/xbmc/blame/00d752bfbe862c67e189d461859e31f471981d33/xbmc/network/upnp/UPnPInternal.cpp#L1158-L1161
+// )
+// is not a standard mime type. according to the specs of UPNP
+// http://upnp.org/specs/av/UPnP-av-ConnectionManager-v3-Service.pdf chapter "A.1.1 ProtocolInfo Definition"
+// "The <contentFormat> part for HTTP GET is described by a MIME type RFC https://www.ietf.org/rfc/rfc1341.txt"
+// all the pre-defined "text/*" MIME by IANA is here https://www.iana.org/assignments/media-types/media-types.xhtml#text
+// there is not any subtitle MIME for now (year 2022), we used to use text/srt|ssa|sub|idx, but,
+// kodi support SUP subtitle now, and SUP subtitle is not really a text(see below), to keep it
+// compatible, we suggest only to math the extension
+//
+// main purpose of this array is to share supported real subtitle formats when kodi act as a UPNP
+// Server or UPNP/DLNA media render
+static const std::string ALLOWED_REAL_SUBTITLE_FORMAT[] = {
```

Comment:
See code guidelines, variables should be camelcase:
https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#87-variables

So, `AllowedRealSubtitleFormat` instead of `ALLOWED_REAL_SUBTITLE_FORMAT`